aioftp/server.py

```diff
@@ -972,7 +973,7 @@ def __init__(self,
         connection = Connection(
             client_host=host,
             client_port=port,
-            server_host=self.server_host,
+            server_host=server_host or self.server_host,
```

Comment:
Wrong order, `self.server_host` should have higher priority I think.

Reply:
At least this change solves the root cause.
@pohmelie, will you accept this PR if it only contains these two lines?

Reply:
Yes, definitely. I had just made some tests and it looks like `writer.transport.get_extra_info("sockname")[0]` return proper ip address for different connection sources (`127.0.0.1` and `192.168.1.43` in my case). Anyway, I can add this lines myself if you do not want make this routine.